modules/ROOT/pages/errors/all-errors.adoc

```diff
@@ -260,6 +279,9 @@ This is a complete list of all transient error status codes Neo4j may return, an
 | Status Code
 | Description
 
+| Neo.TransientError.ChangeDataCapture.FutureIdentifier
+| Change identifier points to a future transaction that has not yet happened on this database instance.
```

Comment:
For change data capture, there was some talk about that we might not want to show it while the feature is still in beta. Better double check with @ali-ince or @l-heemann

Reply:
Thanks @Lojjs for the heads up.

CDC is going into a public beta phase with 5.13 (feature flag is now on by default, as of https://github.com/neo-technology/neo4j/pull/22245). If these changes will be made available with 5.13 - I think we can leave this here, otherwise I think it makes more sense to extract these into a separate PR that's aligned with 5.13.

Reply:
Yes I am thinking it is probably better to add all CDC status codes in a separate PR. Do you have some CDC wall were I can put acardabout it?

Reply:
I think the best place would be cypher operations inbox for that.